sympy/stats/crv_types.py

```diff
@@ -397,6 +399,97 @@ def Beta(name, alpha, beta):
 
     return rv(name, BetaDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+# Noncentral Beta distribution ------------------------------------------------------------
+
+
+class BetaNoncentralDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'beta', 'lamda')
+
+    set = Interval(0, 1)
+
+    @staticmethod
+    def check(alpha, beta, lamda):
+        _value_check(alpha > 0, "Shape parameter Alpha must be positive.")
+        _value_check(beta > 0, "Shape parameter Beta must be positive.")
+        _value_check(lamda >= 0, "Noncentraility parameter Lambda must be positive")
+
+    def pdf(self, x):
+        alpha, beta, lamda = self.alpha, self.beta, self.lamda
+        k = Dummy("k")
+        return Sum(exp(-lamda / 2) * (lamda / 2)**k * x**(alpha + k - 1) *\
+             (1 - x)**(beta - 1) / (factorial(k) * beta_fn(alpha + k, beta)), (k, 0, oo))
```

Comment:
Continued line markers `\` are not required when enclosed in braces.
```diff
-        return Sum(exp(-lamda / 2) * (lamda / 2)**k * x**(alpha + k - 1) *\
-             (1 - x)**(beta - 1) / (factorial(k) * beta_fn(alpha + k, beta)), (k, 0, oo))
+        return Sum(exp(-lamda/2)*(lamda/2)**k * x**(alpha + k - 1) * (1 - x
+             )**(beta - 1)/(factorial(k)*beta_fn(alpha + k, beta)), (k, 0, oo))
```

Reply:
Done. Thanks!